src/quest.coffee

```diff
@@ -226,6 +227,15 @@ class Quest
     else
       console.log "Nothing to output.".gray
 
+  jsonify: (sqlResult, options) ->
+    console.log JSON.stringify row, 4 for row in sqlResult.rows
+
+  outputType: (sqlResult, options) ->
```

Comment:
We're not using the 'options' argument. Also, let's name this 'outputRows' or 'output'. Any particular reason you decided on outputType?